js/providers/Users.js

```diff
@@ -0,0 +1,76 @@
+define(function (require, exports) {
```

Comment:
Why is this separate from `js/modules/WebAPIProvider/UsersAPI.js`? + same comment about `services` folder